### ISSUE TITLE
[BUG] V 2.9.0.6x64 No sound output on Bluetooth speaker

### ISSUE THREAD

**Issue by devmrs** [ramdor/Thetis-2.9.0#101]
I purchased a bluetooth speaker (ANKER SoundCore). It plays anything on the computer except Thetis. I can make it work by sending speaker output to Virtual Cable 1 and then run the MME Repeater to copy sound on Virtual Cable 1 to the Bluetooth speaker.

I tried PowerSDR v 3.4.9 (3/19/18) and it works just fine sending speaker output directly to the bluetooth speaker.

**Comment by ramdor** [comment]
Probably not a bug but some windows audio config or driver issue. Please post over on the apache labs community forum for support. Closing this.

**Comment by devmrs** [comment]
Well I got it to work in a roundabout way. I only have to do this with Thetis -- Not PowerSDR , not Audacity, or any other PC application. 

I made the Bluetooth speaker the default sound output device and in Thetis (and Thetis only) I select Microsoft Sound Mapper Output and I get sound. So, I'm good to go. I'm still not sure why only Thetis works this way, but since it's not a Thetis problem I guess I'll never know.
